manifests/1.1.0/opensearch-1.1.0.yml

```diff
@@ -6,51 +6,51 @@ build:
 components:
   - name: OpenSearch
     repository: https://github.com/opensearch-project/OpenSearch.git
-    ref: "1.1"
+    ref: 1.1.0
```

Comment:
What do you think about updating them with the `commit_id` that we see in manifest.yml that is shipped with the bundle?

Reply:
That could be a viable alternative, and we could eventually do it automatically, but I think the build's intent is better expressed with a tag here, would prefer leaving as is unless there's a compelling argument not to. This is also how we found that a tag was missing on performance-analyzer-rca.

Reply:
Sure. I think we have an issue to cut tags automatically after the release using the commit_id from manifest.yml so I think overall it the same.